accessibility/common-remediations.md

```diff
@@ -103,13 +139,29 @@ We _don’t_ do this:
 <a onclick=”myFunc();”>Visit this location</a>
 ```
 
+### No advance warning when opening new window
+
+#### What's the problem?
+
+Links open in new window without warning the user that this will happen.
+
+#### Who's affected by the problem?
 
-### No advance warning when opening new window 
+* Screen reader users
+* Low vision users who magnify the entire screen
+* Users with cognitive challenges
 
-Avoid opening links in new windows
-This can be disorientating to screen reader users or users with cognitive disabilities.
+#### Why's it a problem?
 
-If you must do it, warn the user before they click on the link that it'll open in a new window. You can use text like "opens in a new window" or a visual icon. If you choose to use an icon, make sure it's accessible to screen reader users.
+Having a window open unexpectedly when you click on a link can be disorientating to users who can only see a portion of the screen, who can't see the screen at all, or who can become easily distracted by unexpected changes to their browsing context. 
+
+#### How do I fix it? 
+
+Avoid opening links in new windows wherever possible. 
```

Comment:
Should we specify that this related to new tabs as well as windows? (as it's browser dependent what `_blank` does)

Reply:
```suggestion
Avoid opening links in new tabs or windows wherever possible. 
```